### PR TITLE
add open, close, events commands

### PR DESCRIPTION
**Pull request by vdbg**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -54,6 +56,15 @@ def get_user(self, id: int) -> BotUser:
         return self.users[id]
 
 
+class Device:
+    def __init__(self, device: dict):
+        self.id: int = int(device["id"])
```
Comment (Nickolaim) [inline main.py:61]: Inconsistent quotes. Format code with `black`.

File: main.py [hunk 2]
```diff
@@ -54,6 +56,15 @@ def get_user(self, id: int) -> BotUser:
...
+        self.label: str = device['label']
+        self.type: str = device['type']
+        self.commands: list[str] = device['commands'] or []
```
Comment (Nickolaim) [inline main.py:64]: I am not sure what's the intention of `device['commands'] or []`, but if there is no `'commands'`, the dictionary will raise KeyNotFound exception, not return None.  If one wants the other behavior, use `device.get('commands', [])`

File: main.py [hunk 3]
```diff
@@ -54,6 +56,15 @@ def get_user(self, id: int) -> BotUser:
         return self.users[id]
 
 
+class Device:
```
Comment (Nickolaim) [inline main.py:59]: Looks nice!

File: main.py [hunk 4]
```diff
@@ -263,19 +272,39 @@ def command_device_status(self, update: Update, context: CallbackContext) -> Non
...
+
+            text.append(row("date", "name", "value"))
+
+            for event in events:
+                text.append(row(event['date'], event['name'], event['value']))
+
+            text.append("```")
+
+            self.send_md(update, context, text)
+            print(events)
```
Comment (Nickolaim) [inline main.py:301]: `print`? No logging?

File: main.py (hunk 3 above)
Comment (vdbg) [reply main.py:59]: indeed

Comment (vdbg) [reply main.py:59]: indeed

File: main.py (hunk 4 above)
Comment (vdbg) [reply main.py:301]: oopsie. fixed

File: main.py (hunk 1 above)
Comment (vdbg) [reply main.py:61]: I'll consider in a separate change

File: main.py (hunk 2 above)
Comment (vdbg) [reply main.py]: looks like it's not needed. removing

**Pull request merged**